docs/design_proposals/event-api-v2.md

```diff
@@ -0,0 +1,119 @@
+# Title
+
+* Author: Nick Kubala
+* Design Shepherd: Any volunteers? :)
+* Date: 04/11/19
+* Status: [Reviewed/Cancelled/Under implementation/Complete]
+
+
```

Comment:
```suggestion
## Final decision

After multiple days invested in this direction we still feel that it is not worthwhile to go down this direction: the complexity of adding go channels adds a lot of complexity and the design more error prone for subtle bugs. The current arguments for going towards this direction is the Event API redesign to centralize event management in the Runner. This in itself is a valuable idea but at this point the current design is simpler to reason about this new design with the go channels, plus if we want to extend the events sent by builders, e.g. progress bar events, we will have to send data through the go channels anyway that represents those events and convert them to events, or directly call the `event` package - at which point the complexity of "handling events from the builders" is going to be the same. 
I am open to introduce a streaming/reactive design in Skaffold on the long run if we have a strong enough argument/featureset for it. For this redesign I don't see that it is worthwhile. I am closing this and the related PRs. 

Thank you @nkubala  and @tejal29  for all the hard work in exploring this avenue in detail in code with testing and thoughtful conversations! 
```

Reply:
I'm just checking if I follow correctly:
- the solution that was discussed and rejected was go channels
- the initial design with bulk updates is out of question because it is not real time
- the favored approach is the as-is design where builders directly call the `event` package

And I think a "than" is missing:
--- simpler to reason about this new
+++ simpler to reason about than this new